incubating/service-now/step.yaml

```diff
@@ -27,11 +27,44 @@ metadata:
           type: service-now
           stage: deploy
           arguments:
-            action: createCR
+            CR_ACTION: createCR
             SN_INSTANCE: https://instance.service-now.com
             SN_USER: ${{SN_USER}}
             SN_PASSWORD: ${{SN_PASSWORD}}
-            cr_body: '{"short_description":"This is a Change Request created by Codefresh","description":"long description for Codefresh build ${{CF_BUILD_ID}}","justification":"because our app is awesome"}'
+            CR_DATA: {"short_description":"This is a Change Request created by Codefresh","description":"long description for Codefresh build ${{CF_BUILD_ID}}","justification":"because our app is awesome"}
+        wait:
+          stage: deploy
+          title: Waiting for the Change Request to be approved
+          type: pending-approval
+          timeout:
+            duration: 48
+            timeUnit: hours
+            finalState: denied
+          modifyCR:
+            title: Close the CR
+            type: service-now
+            stage: deploy
+            arguments:
+              CR_ACTION: closeCR
+              CR_SYSID: ${{CR_SYSID}}'
+              SN_INSTANCE: https://instance.service-now.com
+              SN_USER: ${{SN_USER}}
+              SN_PASSWORD: ${{SN_PASSWORD}}
+              CR_DATA: {"test_plan":"We ran all the test in the awesome test suite. Test pass rate: ${{TEST_RATE}}"}
```

Comment:
CR_DATA should be single quoted in example or bad parsing will take place.

Reply:
Fixed in the next version